rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -1015,4 +1062,31 @@ public boolean changeStartAt(JobId jobId, String startAt)
         }
         return isJobStartAtChanged;
     }
+
+    @Override
+    public TaskFlowJob getInitialJobContent(JobId jobId)
```

Comment:
Maybe the word original is better, but why you we need to retrieve the original content? I thought you just need to add some extra generic information to an existing job?